source/iostreams.tex

```diff
@@ -7462,6 +7493,255 @@
+The Unicode Standard Version 14.0 - Core Specification, Chapter 3.9.
+\end{itemdescr}
+
+\indexlibraryglobal{vprint_nonunicode}%
+\begin{itemdecl}
+void vprint_nonunicode(string_view fmt, format_args args);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+vprint_nonunicode(stdout, fmt, args);
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibraryglobal{vprint_nonunicode}%
+\begin{itemdecl}
+void vprint_nonunicode(FILE* stream, string_view fmt, format_args args);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\expects
+\tcode{stream} is a valid pointer to an output C stream.
+
+\pnum
+\effects
+Writes the result of \tcode{vformat(fmt, args)} to \tcode{stream}.
+
+\pnum
+\throws
+Any exception thrown by the call to \tcode{vformat}\iref{format.err.report}.
+\tcode{system_error} if writing to \tcode{stream} fails.
+May throw \tcode{bad_alloc}.
+\end{itemdescr}
+
+\rSec2[ostream.formatted.print]{Print}
+
+\indexlibraryglobal{print}%
+\begin{itemdecl}
+template<class... Args>
+  void print(ostream& os, @\exposid{format-string}@<Args...> fmt, Args&&... args);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+If the ordinary literal encoding\iref{lex.charset} is UTF-8, equivalent to:
+\begin{codeblock}
+vprint_unicode(os, fmt.@\exposid{str}@, make_format_args(std::forward<Args>(args)...));
+\end{codeblock}
+Otherwise, equivalent to:
+\begin{codeblock}
+vprint_nonunicode(os, fmt.@\exposid{str}@, make_format_args(std::forward<Args>(args)...));
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibraryglobal{println}%
+\begin{itemdecl}
+template<class... Args>
+  void println(ostream& os, @\exposid{format-string}@<Args...> fmt, Args&&... args);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+print(os, "{}\n", format(fmt, std::forward<Args>(args)...));
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibraryglobal{vprint_unicode}%
+\indexlibraryglobal{vprint_nonunicode}%
+\begin{itemdecl}
+void vprint_unicode(ostream& os, string_view fmt, format_args args);
+void vprint_nonunicode(ostream& os, string_view fmt, format_args args);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Behaves as a formatted output function\iref{ostream.formatted.reqmts}
+of \tcode{os}, except that:
+\begin{itemize}
+\item
+failure to generate output is reported as specified below, and
+\item
+any exception thrown by the call to \tcode{vformat} is propagated
+without regard to the value of \tcode{os.exceptions()} and
+without turning on \tcode{ios_base::badbit} in the error state of \tcode{os}.
+\end{itemize}
+After constructing a \tcode{sentry} object,
+the function initializes an automatic variable via
+\begin{codeblock}
+string out = vformat(os.getloc(), fmt, args);
+\end{codeblock}
+If the function is \tcode{vprint_unicode} and
+\tcode{os} is a stream that refers to a terminal capable of displaying Unicode
+which is determined in an implementation-defined manner,
```

Comment:
```suggestion
which is determined in an \impldef{determination of a stream as a Unicode-capable terminal} manner,
```

Reply:
I believe it is correct as is - this is not defining implementation-defined behavior; we're saying "if it is implementation-defined then ...".